src/main/java/com/microsoft/graph/tasks/LargeFileUploadResult.java

```diff
@@ -0,0 +1,22 @@
+package com.microsoft.graph.tasks;
+
+import javax.annotation.Nullable;
+
+/**
+ * Respresents the result of a large file upload task.
+ *
+ * @param <ResultType> type of the deserialized response.
+ */
+public class LargeFileUploadResult<ResultType> {
```

Comment:
Related to this PR, we should expose response headers on the result. Perhaps this is a backlog item.

Reply:
@MIchaelMainer 
Why do you think we'll need to expose other headers? I thought just the location one was required. I know you were off those days but don't hesitate to go comment additional things we might have missed here https://github.com/microsoftgraph/msgraph-sdk-design/pull/42/